src/test/java/edu/kis/vh/nursery/RhymersDemo.java

```diff
@@ -6,7 +6,7 @@
 class RhymersDemo {
 
     public static void main(String[] args) {
-        Rhymersfactory factory = new DefaultRhymersFactory();
+        final Rhymersfactory factory = new DefaultRhymersFactory();
 
         DefaultCountingOutRhymer[] rhymers = {factory.getStandardRhymer(), factory.getFalseRhymer(),
                 factory.getFIFORhymer(), factory.getHanoiRhymer()};
```

Comment:
Below this line, you can find 3 constants such as: 15, 3, 20. You can extract a constant and give those numbers a meaningful name.

Reply:
`        for (int i = 1; i < 15; i++)
            for (int j = 0; j < 3; j++)
                rhymers[j].countIn(i);
        java.util.Random rn = new java.util.Random();
        for (int i = 1; i < 15; i++)
            rhymers[3].countIn(rn.nextInt(20));
        for (int i = 0; i < rhymers.length; i++) {
            while (!rhymers[i].callCheck())
                System.out.print(rhymers[i].countOut() + "  ");
            System.out.println();
        }
        System.out.println("total rejected is "
                + ((HanoiRhymer) rhymers[3]).reportRejected());`
I meant this part of code

Reply:
https://github.com/Cfnatic7/pio_git_rhymers/blob/fdd5030b8ef73241ab83f0455dce17e6946d087c/src/test/java/edu/kis/vh/nursery/RhymersDemo.java#L16-L29
I meant this part of code.